packages/atomic/src/components/search/facets/facet-value-link/facet-value-link.tsx

```diff
@@ -17,7 +17,9 @@ export const FacetValueLink: FunctionalComponent<FacetValueProps> = (
     <li key={props.displayValue} class={props.class}>
       <Button
         style="text-neutral"
-        part={props.part ?? 'value-link'}
+        part={
+          props.part ?? `value-link${props.isSelected ? ' selected-value' : ''}`
```

Comment:
Why added selected-value when we can access the aria-pressed value?

Reply:
Parts are used to access some element from the outside of the web component, it's not doable with aria-pressed AFAIK